File: Eulerpy/Euler5.py
#!/usr/bin/python
import math

upper_bound = math.factorial(20)
divisible = True
for i in xrange(2520, upper_bound, 2520):
	# This is a comment

	for j in xrange(11, 20, 1):
		if i%j != 0:
			divisible = False
			break
		else:
			divisible = True
	if divisible:
		print i
		break

File: Eulerjl/Euler5.jl
upperbound = 1
for i in 2:20
	upperbound *= i
end

function is_div(x)
	for k = 2:20
		if x % k != 0
			return false
		end
	end
	return true
end

j = 2520
while j < upperbound
	if is_div(j)
		println(j)
		break
	end
	j += 2520
end



